route.go

```diff
@@ -82,12 +82,27 @@ func (r *Route) Fallback(handler httpcontract.HandlerFunc) {
 func (r *Route) GlobalMiddleware(middlewares ...httpcontract.Middleware) {
 	timeout := time.Duration(r.config.GetInt("http.request_timeout", 3)) * time.Second
 	defaultMiddlewares := []httpcontract.Middleware{
-		Cors(), Tls(), Timeout(timeout),
+		Cors(), Tls(), Timeout(timeout, nil),
 	}
 	middlewares = append(defaultMiddlewares, middlewares...)
 	r.setMiddlewares(middlewares)
 }
 
+func (r *Route) Recover(recoverFunc func(ctx *gin.Context, err interface{})) {
+    r.instance.Use(func(ctx *gin.Context) {
```

Comment:
There is a `CustomRecovery` method in Gin, we can use it directly.

<img width="1019" alt="image" src="https://github.com/user-attachments/assets/a8e88aed-e56d-41f8-83e0-f3d8679bee0d">

Reply:
will we end up using gin.CustomRecovery?

Reply:
If it can be used directly, yes, we can.

Reply:
Is `CustomRecovery` suitable, please?